### PR TITLE
Parser： Adjust the syntax error log

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

File: parser/parser.y [hunk 1]
```diff
@@ -953,8 +953,7 @@ CreateDatabaseStmt:
 
 		ok := charset.ValidCharsetAndCollation(cs, co)
 		if !ok {
-			//TODO: return error
-			fmt.Println("get charset error")
+			yylex.(*lexer).err("", "get charset error")
```
Comment (shenli) [inline parser/parser.y]: '"Unknown character set  %s or collate %s ", cs, co' is better.

Comment (zimulala) [inline parser/parser.y]: I got it.

**Comment by shenli** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**